### ISSUE TITLE
Pagination: Links not distinguished on their own, link/current states  (2.4.4, 4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#858]
### Expected behavior
- Link's purpose is distinguishable when they stand alone or have programmatic context to make them distinguishable. 
- Link which isn't active is disabled (i.e., currently selected)
- Currently selected "page" communicated in text or programmatically 
### Actual behavior
- Links labels are previous, 1, 2, etc.  don't have any context to their purpose. When there are multiple sets of pagination on the page, this is even more problematic. 
- Currently selected link is an active link which can be clicked, tabbed to, read out as a link
### Resources

See https://mikewest.org/2010/02/an-accessible-pagination-pattern for an example of addressing some of these. Bootstrap uses a nav element with a label as well to be a useful signpost for this type of component when there are multiple on a page and give it a specific label for the types of items being paged through. These would help give the pagination links some context. http://getbootstrap.com/components/#pagination

**Comment by Blackbaud-PatrickOFriel** [comment]
This might be another change we have to push to SKY UX 2 because we're currently just wrapping a ui-bootstrap control

**Comment by Blackbaud-PatrickOFriel** [comment]
Looking at the ui-bootstrap control a bit closer, there might be a way to override their template to address some of these problems. We can take a look. See http://angular-ui.github.io/bootstrap/versioned-docs/1.2.5/#pagination for reference
